README.md

```diff
@@ -101,11 +101,10 @@ GRANT SELECT,INSERT,DELETE,UPDATE ON `vexim`.* to "vexim"@"localhost"
     IDENTIFIED BY 'CHANGE';
 FLUSH PRIVILEGES;
 ```
-Replace the word "CHANGE" with a secure password - It will be used both by the Exim MTA and the web interface to access the database.
-
-If the crypt() function on your system produces DES hashes, you will also have to uncomment the appropriate lines, as noted by the comments, at the end of mysql.sql.
+Replace the word "CHANGE" with a secure password - It will be used both by the Exim MTA and the web interface to access the database. vexim uses SHA512-hashes for user passwords by default which is supported by current systems. You will find alternate hashing shemes (blowfish for *BSD-system or clear-text passwords) at the end of mysql.sql.
```

Comment:
Its Vexim, not vexim. Also, it should probably say "*BSD systems", and I think clear-text should probably not be mentioned at all.